include/libtorrent/alert_types.hpp

```diff
@@ -117,7 +117,7 @@ namespace libtorrent
 		virtual std::string message() const override;
 
 		// The peer's IP address and port.
-		tcp::endpoint const ip;
+		tcp::endpoint const endpoint;
```

Comment:
yeah. to rename these fields you have to keep deprecated duplicates to stay API compatible